androidApp/src/main/java/com/example/tasky/android/agenda/screen/AgendaScreen.kt

```diff
@@ -47,8 +52,19 @@ fun NavGraphBuilder.agendaScreen() {
 @Serializable
 object Agenda
 
+data class AgendaScreenState(
+    val agendas: ImmutableList<AgendaItem> = persistentListOf(),
+    val name: String = "",
+    val startDate: Instant = Clock.System.now(),
+    val numberOfDateShown: Int = 6,
```

Comment:
If this is going to be a static value, it could be a constant

```
class AgendaViewModel {
  companion object {
    private const val DEFAULT_DAYS_TO_SHOW = 6
  }
 }
```